addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -16,7 +16,13 @@ def fill_website_pages(env):
         INSERT INTO website_page
             (url, view_id, website_indexed, website_published, create_uid,
              create_date, write_uid, write_date)
-        SELECT COALESCE(wm.url, '/page/' || iuv.name), iuv.id, TRUE, TRUE,
+        SELECT COALESCE(
+                wm.url,
+                '/page/' || COALESCE(
+                    substring(iuv.key from 9), imd.name, iuv.name
```

Comment:
```suggestion
                    substring(iuv.key from (strpos(iuv.key, '.')+1)), imd.name, iuv.name
```

Reply:
thanks, but I don't believe this fixes the issue you mentioned. when we had an xmlid as key in v10, it was a full xmlid in the url. So if we want to cover this case too, I think we better add another left join that handles the non-website xmlids/keys